public/stmyec_wcm_2022/meta_study.txt

```diff
@@ -0,0 +1,8 @@
+type_of_cancer: stmyec
+cancer_study_identifier: stmyec_wcm_2022
+name: Myoepithelial Carcinomas of the Soft Tissue (CSH Molecular Case Studies, 2022)
```

Comment:
Can you change the name to  Myoepithelial Carcinomas of the Soft Tissue (WCM, CSH Molecular Case Studies 2022)

Reply:
Thanks. Will update shortly.

Reply:
@ritikakundra : Name changed